lib/modules/manager/bundler/range.ts

```diff
@@ -1,22 +0,0 @@
-import type { RangeStrategy } from '../../../types';
-import type { RangeConfig } from '../types';
-
-/*
- * The getRangeStrategy() function is optional and can be removed if not applicable.
- * It is used when the user configures rangeStrategy=auto.
- *
- * For example in npm, when rangeStrategy is auto we:
- *  - Always pin "devDependencies"
- *  - Pin "dependencies" only if we detect that it's probably an app not a library
- *  - Always widen "peerDependencies"
- *
- * If this function is not present then the default 'replace' value will be used.
- *
- */
-export function getRangeStrategy(config: RangeConfig): RangeStrategy {
-  if (config.rangeStrategy === 'auto') {
```

Comment:
is this case now handled somewhere else?

Reply:
It was already redundant. Renovate falls back to replace if no getRangeStrategy is specified